docs/application/native/get-started/wearable/first-app.md

```diff
@@ -284,51 +284,54 @@ For more information on using the emulator features, see [Using Emulator Control
 To run the application on a target device:
 
 1.  Connect the wearable target device to your computer:
-    1.  Define settings on the device:
-        -   Go to **Settings \> Connections**, and switch on Bluetooth.
 
-            ![Switch on Bluetooth](media/emulator_target_bt.png)
+    1. Define settings on the device:
 
-            ![Switch on Bluetooth](media/emulator_target_bt2.png)
+        -   Execute the **Settings** application.
 
-        -   Go to **Settings \> Connections**, and switch on Wi-Fi.
+            ![Settings](media/wearable_settings_app.png)
+
+        -   Go to **Connections > Wi-Fi**, and switch on Wi-Fi.
+
+            ![Enable Wi-Fi](media/testing_your_app_on_gear3.png) ![Select Wi-Fi AP](media/testing_your_app_on_gear4.png)
+
+        -   Go to **Wi-Fi networks > Scan**.
+
+            ![Enable Wi-Fi](media/testing_your_app_on_gear5.png) ![Select Wi-Fi AP](media/testing_your_app_on_gear6.png)
 
             The device and the computer must be connected to the same Wi-Fi network.
 
-            Note the IP address the device is using.
+        -   Note the IP address the device is using.
 
-            ![Switch on Wi-Fi](media/emulator_target_wifi.png)
+            ![IP Address](media/testing_your_app_on_gear7.png)
 
-        -   Go to **Settings \> Gear info**, and switch on the debugging mode.
+        -   Go to **About watch > Debugging**, and switch on the debugging mode.
 
-            ![Switch on debugging](media/emulator_target_debug.png)
+            ![Switch on debugging](media/testing_your_app_on_gear1.png) ![Debugging menu](media/testing_your_app_on_gear2.png) 
 
     <a name="remote_device"></a>
 
     2.  Use the Remote Device Manager to connect the wearable device:
+
         1.  In the **Device Manager**, launch the Remote Device Manager by clicking the related icon.
 
             ![Launch the Remote Device Manager](media/remote_conn_mgr.png)
 
-        2.  In the **Remote Device Manager** window, click **+**.
-
-            ![Add new device](media/remote_conn_new_ww.png)
-
-        3.  In the **Add Device** window, enter the device and network details (use the IP address you noted before), and click **Add**.
+        2.  In the **Remote Device Manager** window, click the **Scan** icon(![Scan icon](media/remote_conn_scan_icon.png)).
 
-            ![Define device details](media/remote_conn_add_ww.png)
+            ![Scan devices](media/remote_conn_scan.png)
 
-        4.  In the **Remote Device Manager** window, switch the new device on by clicking the switch under **Connect**.
+        3.  In the **Remote Device Manager** window, switch the new device on by clicking the switch under **Connect**.
```

Comment:
switch under **Connect**.

In the image, it is **Connection**.
![image](https://user-images.githubusercontent.com/9958374/68450911-0abbff80-0230-11ea-87b0-f6d4af95665f.png)